css/index.css

```diff
@@ -1 +1,468 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+    display: block;
+}
+body {
+    line-height: 1;
+}
+ol, ul {
+    list-style: none;
+}
+blockquote, q {
+    quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content: '';
+    content: none;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+    font-size: 62.5%;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', sans-serif;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+/* Body */
+
+.wrapper {
+    max-width: 880px;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+p, h2 {
+    font-size: 1.6rem;
+}
+
+nav, header, section {
+    display: flex;
+    padding: 34px 0px 0px;
+}
+
+/* Disabled links */
+
+nav a, header a, section a {
+    text-decoration: none;
+}
+
+header a, section a {
+    color: black;
+}
+
+
+/*Navigation */
+
+nav {
+    justify-content: space-between;
+    
+}
+.nav {
+    display: flex;
+    flex-wrap: wrap;
+}
+nav a {
+    color: #bebebe;
+    padding: 0px 30px;
+    align-self: center;
+    font-weight: bold;
+    font-size: 1.6rem;
+}
+nav a:hover {
+    color: black;
+}
+.menu a:first-child {
+    padding-left: 0px;
+}
+.logo a {
+    padding: 0px;
+}
+
+/* Header */
+
+header {
+    border-bottom: 1.5px solid black;
+    padding-bottom: 70px;
+}
+
+header a {
+    border: 1px solid black;
+    margin: 0px 127px 0px;
+    text-align: center;
+    display: flex;
+    font-weight: bold;
+    flex-flow: column;
+    padding: 8px;
+    font-size: 1.6rem;
+}
+header a:hover {
+    background-color: black;
+    color: #fff;
+}
+
+/* Content */
+
+section img {
+    width: 100%;
+    padding-bottom: 20px;
+}
+section a {
+    padding-bottom: 5px;
+    display: inline-block;
+}
+
+/* Footer */
+
+footer {
+    text-align: center;
+    padding-bottom: 30px;
+    font-size: 1.6rem;
+}
+
+
+/* Main Page CSS */
+
+.main header h1 {
```

Comment:
I think that I already gave you feedback on centering the words in this header, on Responsive Design I. So review that. 

Also, it looks like the heading text centering breaks in your tablet view such that the rows move independently with the size of the viewport.